tutorial.md

```diff
@@ -36,7 +36,7 @@ This section will walk you through installing the following components:
 Download the `runc` release binary:
 
 ```
-wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc2/runc-linux-amd64
+wget https://github.com/opencontainers/runc/releases/download/v1.0.0-rc3/runc-linux-amd64
```

Comment:
this isn't fully compatible with CRI-O - we do need a particular version of runc ahead of rc3, the commit we need is `639454475cb9c8b861cc599f8bcd5c8c790ae402`. If we don't have that commit when we `delete -f` a container which doesn't exist return an error. By using `639454475cb9c8b861cc599f8bcd5c8c790ae402` we don't get the error, which is the right behavior.

I think we should drop this section and just warn that we need at least runc to be built at least from `639454475cb9c8b861cc599f8bcd5c8c790ae402`.